diff -urN libappindicator-0.4.1.old/src/app-indicator.c libappindicator-0.4.1/src/app-indicator.c
--- libappindicator-0.4.1.old/src/app-indicator.c	2011-09-28 18:40:56.663816000 +0100
+++ libappindicator-0.4.1/src/app-indicator.c	2011-10-05 21:01:36.389664639 +0100
@@ -33,8 +33,8 @@
 
 #include <libdbusmenu-glib/menuitem.h>
 #include <libdbusmenu-glib/server.h>
-#include <libdbusmenu-gtk/client.h>
-#include <libdbusmenu-gtk/parser.h>
+#include <libdbusmenu-gtk3/client.h>
+#include <libdbusmenu-gtk3/parser.h>
 
 #include <libindicator/indicator-desktop-shortcuts.h>
 
